packages/terra-clinical-app-delegate/tests/jest/AppDelegate.test.jsx

```diff
@@ -0,0 +1,263 @@
+import AppDelegate from '../../src/app-delegate/AppDelegate';
+
+describe('propType', () => {
+  it('should provide a defined PropType', () => {
+    expect(AppDelegate.propType).not.toBe(undefined);
+  });
+});
+
+describe('create', () => {
+  it('should expose a create function that creates an AppDelegate instance', () => {
+    const disclose = () => {};
+    const dismiss = () => {};
+    const closeDisclosure = () => {};
+    const goBack = () => {};
+    const maximize = () => {};
+    const minimize = () => {};
+    const notSupported = () => {};
+
+    const appDelegateInstance = AppDelegate.create({
+      disclose, dismiss, closeDisclosure, goBack, maximize, minimize, notSupported,
+    });
+
+    expect(appDelegateInstance.disclose).toBe(disclose);
+    expect(appDelegateInstance.dismiss).toBe(dismiss);
+    expect(appDelegateInstance.closeDisclosure).toBe(closeDisclosure);
+    expect(appDelegateInstance.goBack).toBe(goBack);
+    expect(appDelegateInstance.maximize).toBe(maximize);
+    expect(appDelegateInstance.minimize).toBe(minimize);
+    expect(appDelegateInstance.notSupported).toBe(undefined);
```

Comment:
That's just a test attribute to ensure that we only set the functions we're expecting. We don't want any old attribute to be set on these things. 

I can probably change that to be more obviously test code.